lib/conjur/fpm/package.sh

```diff
@@ -34,21 +30,29 @@ bundle_clean
 if [ `ls | wc -l` -eq 0 ]; then
   echo No dev dependencies, skipping dev package
 else
-  fpm -s dir -t deb -n conjur-$project_name-dev -v $version -C . \
+  for file_type in deb rpm
+  do
+    echo "Building conjur-$project_name-dev $file_type package"
+
+    fpm \
+    -s dir \
+    -t $file_type \
+    -n conjur-$project_name-dev \
+    -v $version \
+    -C . \
     --maintainer "Conjur Inc." \
```

Comment:
Maybe it's worth changing `Conjur Inc.` to `CyberArk`? Don't want to increase your scope, only if you feel comfortable doing that.
Same with the url, from `https://www.conjur.net` to `https://www.cyberark.com`

Reply:
what should we do with this?
```
  spec.authors       = ["Kevin Gilpin"]
  spec.email         = ["kgilpin@conjur.net"]
```

@izgeri do you have a suggestion?

Reply:
oof let's definitely update this

authors: CyberArk Software, Inc.
email: conj_maintainers@cyberark.com

Reply:
thanks :)